modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties

```
@@ -500,6 +500,10 @@ dimensions=Dimensions
 discount-amount=Discount Amount
 discount-application-strategy=Discount Application Strategy
 discount-level=Discount Level
+discount-level-1=L1
+discount-level-2=L2
+discount-level-3=L3
+discount-level-4=L4
```

@andrea-ale-sbarra can we make it like this so the keys and values match?
```suggestion
l1=L1
l2=L2
l3=L3
l4=L4
```